--- configure.ac.original	2006-11-09 16:21:26.000000000 +1100
+++ configure.ac	2007-10-22 17:31:04.000000000 +1000
@@ -35,4 +35,5 @@
 AC_CONFIG_FILES([Makefile include/Makefile library/Makefile library/t/Makefile module/Makefile module/apache2/Makefile glue/Makefile])
 AC_CONFIG_FILES([build/doxygen.conf include/groups.dox])
 AC_CONFIG_FILES([apreq2-config], [chmod +x apreq2-config])
+AC_CONFIG_FILES([libapreq2.pc])
 AC_OUTPUT
